kitsune/flagit/views.py

```diff
@@ -97,6 +104,35 @@ def flagged_queue(request):
     )
 
 
+@login_required
+@permission_required("flagit.can_moderate")
+def moderate_content(request):
+    """Display flagged content that needs moderation."""
```

Comment:
Maybe you're planning this in a subsequent PR, but I think we should either create a new permission for topic moderation that we give to users who understand the taxonomy, or else add another check here that the user is a member of a specific group that we define for that purpose.

Reply:
I am going to combine this with the locking PR